helios-services/pom.xml

```diff
@@ -319,6 +318,13 @@
       <version>0.8</version>
     </dependency>
 
+    <!-- versions that we have to manage to avoid requireUpperBoundDeps violations -->
```

Comment:
You mean httpclient? Yea I was going to do this originally but hesitated because I didn't want to change too much. I'll move it.